### PR TITLE
Allow forks

### PR DESCRIPTION
**Pull request by jdel**
Reverts Reverts #3 and fix NPM dependencies that use `ssh://`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 ARG FFRK_PROXY_VERSION=master
 
 RUN apk update \
- && apk add --no-cache ca-certificates curl \
+ && apk add --no-cache ca-certificates curl git \
  && apk upgrade \
- && rm -rf /var/cache/apk/* \
- && curl -L https://github.com/ThauEx/ffrk-proxy/archive/${FFRK_PROXY_VERSION}.tar.gz | gunzip | tar -xf - -C / \
+ && git config --global url."https://".insteadOf git:// \
+ && git config --global url."https://".insteadOf ssh:// \
```
Comment (jarrettone) [inline Dockerfile]: @jdel where'd you find this? This is super slick and I'm definitely stealing it for my bag o' tricks!

Comment (jdel) [reply Dockerfile]: Couple of minutes of googling got me here: https://gist.github.com/taoyuan/bfa3ff87e4b5611b5cbe

**Comment by jdel** [comment]
So, that branch builds for me now, with your fork, you can try with:

```bash
docker build -t jdel/ffrk-proxy:jarrettone --build-arg FFRK_PROXY_FORK=jarrettone/ffrk-proxy github.com/jdel/docker-ffrk-proxy.git#allow-fork
```

**Comment by jdel** [comment]
Let me know if this works for you. 

In the meantime, I will use this branch to troubleshoot some deployment issues with bintray.

**Comment by jarrettone** [comment]
It should, but I won’t get a chance to test until, potentially, next week.

**Comment by jdel** [comment]
Hi @jarrettone, did you have a chance to check so I can merge this ?

**Comment by jarrettone** [comment]
@jdel works perfectly to build a container that will run. Unfortunately, there seems to be a new issue in ffrk-proxy that prevents me from using the container successfully. I believe this is related to IPv6 configuration on either the device or the container. If this works for you, go ahead and merge. I’ll be trying to track down the connectivity issue and will hopefully submit that to upstream. I doubt any changes will be needed here.

**Comment by jdel** [comment]
I haven't been using ffrk proxy for a while now, so I will merge for now and see about the connectivity issue in an other PR.

**Pull request merged**